de.metas.business/src/main/java-legacy/org/compiere/model/CalloutInOut.java

```diff
@@ -176,7 +177,7 @@ public String docType(final ICalloutField calloutField)
 		final I_M_InOut inout = calloutField.getModel(I_M_InOut.class);
 		final IDocumentNoInfo documentNoInfo = Services.get(IDocumentNoBuilderFactory.class)
 				.createPreliminaryDocumentNoBuilder()
-				.setNewDocType(inout.getC_DocType())
+				.setNewDocType(load(inout.getC_DocType_ID(), I_C_DocType.class))
```

Comment:
```suggestion
				.setNewDocType(loadOutOfTrx(inout.getC_DocType_ID(), I_C_DocType.class))
```
i think in this case it's pretty safe; doctype is certainly not altered in this transaction